src/@types/event.ts

```diff
@@ -81,6 +81,7 @@ export enum EventType {
     PushRules = "m.push_rules",
     Direct = "m.direct",
     IgnoredUserList = "m.ignored_user_list",
+    IgnoredInvites = "org.matrix.msc3840.ignored_invites",
```

Comment:
we're not supposed to define unstable event types here - we should be using `UnstableValue().name`